sandboxed_containers/sandboxed-containers-release-notes.adoc

```diff
@@ -0,0 +1,171 @@
+=== Enhanced logging
+
+Administrators can now collect enhanced logs for {sandboxed-containers-first} runtime components. Enhanced logs are available when the CRI-O log level is set to `debug`. These logs are collected by the must-gather tool, or can be viewed in the node journal.
+//For more information, see xref:../sandboxed_containers/troubleshooting-sandboxed-containers.adoc#sandboxed-containers-enable-debug-logs_troubleshooting-sandboxed-containers[Enabling debug logs for {sandboxed-containers-first}].
+
+[id="node-eligibility-rn"]
+=== Check node eligibility to run {sandboxed-containers-first}
+
+Administrators can now check the eligibility of cluster nodes to run {sandboxed-containers-first}. This feature uses the Node Feature Discovery (NFD) operator to detect  node capabilities. Eligible nodes are labeled with `feature.node.kubernetes.io/runtime.kata`, and the {sandboxed-containers-operator} uses this label to select candidate nodes for installation.
+
+The administrator must deploy the NFD operator to use this feature, create a specific *NodeFeatureDiscovery* custom resource, and enable `checkNodeEligibility' when creating the *KataConfig* custom resource.
+//For more information, see xref:../sandboxed_containers/deploying-sandboxed-container-workloads.adoc#sandboxed-containers-check-nodes__deploying-sandboxed-containers[Checking the eligibility of cluster nodes to run {sandboxed-containers-first}].
+
+[id="OSC-compatibility-with-CNV"]
+=== {sandboxed-containers-first} compatibility with {VirtProductName}
+
+Users can now run {sandboxed-containers-first} on clusters with {VirtProductName} when VMs are configured correctly.
+//For more information, see xref:../sandboxed_containers/understanding-sandboxed-containers.adoc#sandboxed-containers-with-other-ocp-components_understanding-sandboxed-containers[Using {sandboxed-containers-first} with {VirtProductName}]
+
+[id="support-AWS-bare-metal"]
+=== {sandboxed-containers-first} availability on AWS bare-metal (tech preview)
+
+Users can now install {sandboxed-containers-first} on AWS bare-metal clusters. This feature is in tech preview and not fully supported. For more information, see xref:../sandboxed_containers/understanding-sandboxed-containers.adoc#understanding-sandboxed-containers[Understanding {sandboxed-containers-first}].
+
+[id="sandboxed-containers-1-2-bug-fixes"]
+== Bug fixes
+
+* Previously, using loop devices in a sandboxed container was not possible due to missing kernel modules. With this release, these kernel modules are included in the package. (link:https://issues.redhat.com/browse/KATA-1334[*KATA-1334*])
+
+* Previously, the {sandboxed-containers-operator} bundle was using an incorrect value for the `com.redhat.openshift.versions` label when building the container image. With this release, the Operator is available in the correct operator catalog for {product-title} version {product-version}. (link:https://issues.redhat.com/browse/KATA-1019[*KATA-1019*])
```

Comment:
KATA-1019 was fixed in 1.1 already (and first in 1.0.2 as KATA-1012), do we need to list it here again as a bug fix? I does no harm leaving it in, but IMHO we could also drop it.